src/ansi.rs

```diff
@@ -715,6 +723,44 @@ bitflags! {
     }
 }
 
+/// This only applies to keys corresponding to ascii characters.
+///
+/// For the details on how to implement the mode handling correctly, consult [`XTerm's
+/// implementation`] and the output of XTerm's provided [`perl script`], which output
+/// example could be found [`here`].
```

Comment:
```suggestion
/// For the details on how to implement the mode handling correctly, consult [`XTerm's
/// implementation`] and the output of XTerm's provided [`perl script`], the output of which
/// can be found [`here`].
```

Alternatively whose also works. "an example of its output can be found here" is another option.